src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -545,6 +545,23 @@ bool Eltwise::isWithBroadcast() {
     return false;
 }
 
+void Eltwise::init() {
+    if (m_attrs.data.algo == Algorithm::EltwisePowerStatic && getOriginalInputPrecisionAtPort(0) == ov::element::bf16) {
+        float lowest = static_cast<float>(std::numeric_limits<ov::bfloat16>::lowest());
+        float max = static_cast<float>(std::numeric_limits<ov::bfloat16>::max());
```

Comment:
NIT. Please add `constexpr`
```suggestion
        constexpr float lowest = static_cast<float>(std::numeric_limits<ov::bfloat16>::lowest());
        constexpr float max = static_cast<float>(std::numeric_limits<ov::bfloat16>::max());
```